python/ray/_private/gcs_utils.py

```diff
@@ -208,10 +209,12 @@ def address(self):
         return self._channel._gcs_address
 
     @_auto_reconnect
-    def internal_kv_get(self, key: bytes, namespace: Optional[bytes]) -> bytes:
+    def internal_kv_get(
+        self, key: bytes, namespace: Optional[bytes], timeout: Optional[int] = None
```

Comment:
Should the type annotation be `Optional[float]`? Also I'm not a fan of gRPC's naming of this parameter. Should it be `timeout_s` to indicate the unit?

Reply:
I feel in python whenever there is a timeout, it's always seconds. is it a conventional way to do this in python? I'll change the type to float.

Reply:
For example, 
https://requests.readthedocs.io/en/latest/user/quickstart/#timeouts
https://github.com/pytest-dev/pytest-timeout#usage
https://docs.python.org/3/library/asyncio-task.html#timeouts

Reply:
Sounds good. We can go with the convention.